cumulusci/cumulusci.yml

```diff
@@ -98,6 +102,14 @@ tasks:
             settings_type: EnhancedNotesSettings
             settings_field: IsEnhancedNotesEnabled
             value: True
+    check_rd2_enabled:
+        description: Preflight check to validate that Enhanced Recurring Donations are enabled.
+        group: Salesforce Preflight Checks
+        class_path: cumulusci.tasks.preflight.settings.CheckCustomSettingsValue
+        options:
+            settings_type: npe03__Recurring_Donations_Settings__c
+            settings_field: npsp__IsRecurringDonations2Enabled__c
+            value: True
```

Comment:
I don't think a task config that is preconfigured to check for an NPSP feature belongs in CumulusCI, which is a general-purpose framework designed to be used with many different products. I would define it in cumulusci.yml for the project that needs to perform the check instead.

Reply:
Good point. Removed in ff5c58b7e0cb6ce10660362bc5df05d310feee13